### PR TITLE
adds materials from July meetup

### PR DESCRIPTION
**Pull request by RabeaGleissner**
@NataliaLKB @iteles 

Could one of you ladies review and merge the PR? Thank you!

### CODE REVIEW

File: meetups/2016-july/readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+##### Read [Ina's blog post](http://tsetsova.github.io/events/2016/08/01/tdd-at-ladies-of-code.html) about the Roman Numerals kata and property based testing.
+
+##### Ina also uploaded [her code to Github](https://github.com/tsetsova/roman_numerals).
+
+</br>
+
+### Part 3: TDD beginner's workshop - by [_Natalia Baltazar_](https://twitter.com/NataliaLKB)
+
+Natalia put together a workshop as an introduction to TDD.
```
Comment (NataliaLKB) [inline meetups/2016-july/readme.md]: Change to "introduce" as dwyl created the workshop, I just shared it.

**Comment by NataliaLKB** [comment]
Just one comment, but I can merge after :)

File: meetups/2016-july/readme.md (hunk 1 above)
Comment (RabeaGleissner) [inline meetups/2016-july/readme.md]: Ah... whoops! Giving you too much credit there 😉

**Comment by RabeaGleissner** [comment]
Thanks @NataliaLKB !
I updated the wording and added a link to DWYL as well.

**Pull request merged**